src/lfs/server_receiver.lua

```diff
@@ -48,6 +48,10 @@ local function httpReceiver(sck, payload)
     print("Heap: ", node.heap(), "HTTP: ", "Device")
     response.text(sck, require("server_device")(request))
 
+  elseif request.path == "/zone" then
+    print("Heap: ", node.heap(), "HTTP: ", "Zone")
+    response.text(sck, require("server_zone")(request))
+
```

Comment:
I'm not sure that this new endpoint is needed. Couldn't we just use the same /device endpoint and check for `zone` or `pin` in the query params?

Reply:
I added it for testing with homeassistant where I changed the `KONN_MODEL: CONF_PIN` to `KONN_MODEL: CONF_ZONE`, which then asks for the `/zone` endpoint.

I will remove it now, however it should be noted that this may be required in the future if the `pin` notation is removed and homeassistant updated.

Reply:
done